src/resolver.rs

```diff
@@ -412,6 +443,10 @@ impl AnnotationMap for AnnotationMapImpl {
         self.type_hint_map.get(&s.get_id())
     }
 
+    fn get_function_annotation(&self, s: &AstStatement) -> Option<&AstStatement> {
```

Comment:
like above:
/// returns the function call previously annoted on `s` via `annotate_hidden_function_call(...)`
`get_hidden_function_call(...)`